### DISCUSSION TITLE
JSON export

### DISCUSSION THREAD

**Discussion by devtweakr** [Faltenreich/Diaguard#45] [Ideas]
Is it possible to add JSON export/import, I would like to create a web app equivalent.

Thanx

**Comment by Faltenreich** [comment]
Hi @devtweakr and sorry for the late reply - I completely missed the discussions section until today. 🙈 
Exporting/importing JSON would technically be feasible but mean some effort in order to get it right.
May I ask what you need it for and why CSV does not match your requirements?

**Comment by devtweakr** [comment]
No need anymore, I can use CSV. Thanx.
